sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs

```diff
@@ -33,6 +33,16 @@ public class HealthcareOperation : Operation<RecognizeHealthcareEntitiesResultCo
         /// </summary>
         public override string Id { get; }
 
+        /// <summary>
+        /// Properties for healthcare operation
+        /// </summary>
+        public HealthcareOperationProperties Properties { get; set; }
+
+        /// <summary>
+        /// Gets an Status representing the operation.
+        /// </summary>
+        public JobStatus Status { get; internal set; }
```

Comment:
preferably there should be a private `_status` variable.  That way you don't directly modify the property.
`public JobStatus Status => _status`
See how we do this in other places of this class => https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareOperation.cs#L68